modules/digitaluniverse/rendering/renderabledumeshes.cpp

```diff
@@ -246,9 +246,9 @@ documentation::Documentation RenderableDUMeshes::Documentation() {
 
 RenderableDUMeshes::RenderableDUMeshes(const ghoul::Dictionary& dictionary)
     : Renderable(dictionary)
-    , _alphaValue(TransparencyInfo, 1.f, 0.f, 1.f)
     //, _scaleFactor(ScaleFactorInfo, 1.f, 0.f, 64.f)
```

Comment:
remove commented code

Reply:
I try to avoid poking in code that is not directly related to my changes, so always get a bit confused when I come across commented out code that I don't understand why it's there. I always think "someone probably saved it for a reason" and avoid touching it. 

Is it better to just remove it? (I would love to)

Reply:
I raised the question in core. In this case there seemed to be an unused property which would not be much work to figure out how to reimplement, so I'd say remove it.  Commented code should always be documented. If you come across cases where there are some more complex commented code and you are unsure you can always contact the developer who wrote it.

Example on documentation: 

// TODO WeirdRubberDuck: Keep this code for implementation of x and y because of z
// the commented code

Reply:
Thanks for asking about it. Then I'll make an extra clean upp pass of the files I have changed :)